core/src/server/Config.cpp

```diff
@@ -363,19 +363,25 @@ Config::PrintAll() {
 Status
 Config::CheckServerConfigAddress(const std::string& value) {
     if (!ValidationUtil::ValidateIpAddress(value).ok()) {
-        return Status(SERVER_INVALID_ARGUMENT, "Invalid server config address: " + value);
+        std::string msg = "Invalid server IP address: " + value
+                          + ". Possible reason: server_config.address is invalid in server_config.yaml.";
```

Comment:
Better to uniform the format: xxx_config.xxx in xxx.yaml is invalid
Please also check below messages.